pandas/core/indexing.py

```diff
@@ -1676,7 +1676,6 @@ def isetter(loc, v):
                     ser = v
                 else:
                     # set the item, possibly having a dtype change
-                    ser._consolidate_inplace()
```

Comment:
this one is a no-op since `ser` is a series (I think?), so can easily be removed